classes/submission/ReviewFilesDAO.inc.php

```diff
@@ -76,6 +76,27 @@ function check($reviewId, $fileId) {
 		$result->Close();
 		return $returner;
 	}
+
+	/**
+	 * Select Review file ids by reviewId
+	 * @param $reviewId integer
+	 * @return array
+	 */
+	function getByReviewId($reviewId) {
```

Comment:
The `getBy` convention generally returns an object of the DAO's `newDataObject` function's return type. Since you're returning IDs here, I'd suggest renaming the function.